app/src/androidTest/java/org/mozilla/reference/browser/ui/robots/AddonsManagerRobot.kt

```diff
@@ -61,6 +62,11 @@ class AddonsManagerRobot {
         gotItButton.click()
     }
 
+    fun clickRemoveAddonButton() {
+        removeAddonButton().click()
+        mDevice.waitForIdle()
+    }
```

Comment:
Not sure if we will need a wait for not null or wait for an element before moving on, just to be sure we are where we want..

Reply:
I could change `removeAddonButton()` to `mDevice.findObject(UiSelector().resourceId("$packageName:id/remove_add_on"))`

and `clickRemoveAddonButton` to `removeAddonButton().clickAndWaitForNewWindow()`